src/Components/GEOSldas_GridComp/GEOSlandassim_GridComp/clsm_ensupd_upd_routines.F90

```diff
@@ -1497,6 +1511,7 @@ subroutine get_obs_pred(                                       &
     if (get_rzmc_lH)   rzmc_lH  = rzmc_l
     if (get_tsurf_lH)  tsurf_lH = tsurf_l
     if (get_FT_lH)     FT_lH    = FT_l
+    if (get_asnow_lH)  asnow_lH = asnow_l !jpark50
```

Comment:
I'm honestly confused why we have this block of assignments.  Just above, we unpacked "tile_data_lH" into "*_lH", and now we seem to be overwriting "lH" with the simple local "l" array.  We'll have to take another close look at this later.

Reply:
Comment probably outdated.  Closing for now.